### PR TITLE
fix the test suite broken when run in a non English speaking country

### PR DESCRIPTION
**Pull request by ghost**
Only proof that it works
```
... runnable/literal.d              -fPIC (-inline -release -g -O)
 ... runnable/warning1.d            -w  -fPIC ()
 ... runnable/testsocket.d           -fPIC ()
 ... runnable/test17619.sh
 ... runnable/cdvecfill.sh
 ... runnable/test13666.sh
 ... runnable/test18456.sh
 ... runnable/test12486.d            -fPIC (-inline -release -g -O)
 ... runnable/test13742.sh
 ... runnable/test35.sh
 ... runnable/test13774.sh
 ... runnable/test18868.sh
 ... runnable/test_switches.sh
 ... runnable/test9287.sh
 ... runnable/test18141.sh
 ... runnable/test17968.sh
 ... runnable/test39.sh
 ... runnable/test10386.sh
 ... runnable/test18076.sh
 ... runnable/paranoia.sh
 ... runnable/test16096.sh
 ... runnable/gdb15729.sh
 ... runnable/link14198b.sh
 ... runnable/test15568.d           -unittest -main -O  -fPIC (-inline -release -g)
 ... runnable/test10567.sh
 ... runnable/link14198a.sh
 ... runnable/test2.sh
 ... runnable/link846.sh
 ... runnable/cdcmp.sh
 ... runnable/test18902.sh
 ... runnable/test_shared.sh
 ... runnable/link14834.sh
 ... runnable/test18335.sh
 ... runnable/test9377.sh
 ... runnable/depsprot.sh
 ... runnable/linkdebug.sh
 ... runnable/mixin.sh
 ... runnable/test19655.sh
 ... runnable/test44.sh
 ... runnable/testconst.d            -fPIC (-inline -release -g -O)
 ... runnable/xtest46_gc.d          -lowmem -Jrunnable  -fPIC (-inline -release -g -O)
make[1] : on quitte le répertoire « /home/basile/dev/repos/dlang/dmd/test »
```

### CODE REVIEW

**Comment by jacob-carlborg** [comment]
Have a look at https://github.com/dlang/dmd/pull/9704 as well.

File: test/runnable/cdcmp.sh [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 
+export LC_ALL=en_US
```
Comment (kubo39) [inline test/runnable/cdcmp.sh]: How about using `LC_ALL=C` instead? For example, containers may not have en_US.

Comment (Basile-z) [reply test/runnable/cdcmp.sh]: Yeah but you can suggest it as a change maybe, I have made the PR online. So just after my export sput `export LC_ALL=C`.

Comment (Basile-z) [reply test/runnable/cdcmp.sh]: I can do it online too actually

Comment (Basile-z) [reply test/runnable/cdcmp.sh]: okay done, that works too.

**Review by kubo39 — APPROVED** [review]

**Review by RazvanN7 — APPROVED** [review]

**Pull request merged**